jenkinsci/plot-plugin
# Migrate and improve documentation from old Jenkins Wiki

### What has been done
1. Migrated plugin documentation from [old Wiki](wiki.jenkins.io/display/JENKINS/Plot+Plugin) to GitHub. Read more at [Plugin Documentation-as-Code: Moving docs to GitHub](https://jenkins.io/blog/2019/10/21/plugin-docs-on-github/).
2. Added plugin badges to README using [shields.io](https://shields.io/)
- Predefined `Jenkins Plugins` badge from ["Version"](https://shields.io/category/version) section
![image](https://user-images.githubusercontent.com/3036347/77748262-ba034280-7028-11ea-9e3e-234847437f29.png)
- Predefined `Jenkins Plugin Installs` badge from ["Downloads"](https://shields.io/category/downloads) category
![image](https://user-images.githubusercontent.com/3036347/77748459-11a1ae00-7029-11ea-91fa-b47e18fe2980.png)
- [Dynamic badge for release notes](https://img.shields.io/badge/dynamic/json?color=blue&label=changelog&query=$.tag_name&url=https://api.github.com/repos/jenkinsci/plot-plugin/releases/latest) using GitHub JSON API for fetching latest release version https://api.github.com/repos/jenkinsci/plot-plugin/releases/latest and [JSON Path](https://jsonpath.com/) for extracting version number `$.tag_name`
![image](https://user-images.githubusercontent.com/3036347/77748669-680eec80-7029-11ea-886a-302c0a7b8cc0.png)
3. Mentioned importance of having proper labels in PRs in PR template 

### How to test
1. Make sure content from [old Wiki](wiki.jenkins.io/display/JENKINS/Plot+Plugin) page is fully migrated to README
2. Perform proof-reading of the README to spot any mistakes
3. Make sure shields.io badges are showing correct data and lead to correct pages when clicked

### Checklist
